traffic_portal/app/src/common/modules/form/user/form.user.tpl.html

```diff
@@ -25,6 +25,7 @@
             <li class="active">{{userName}}</li>
         </ol>
         <div class="pull-right" role="group" ng-show="!settings.isNew">
+            <button name="viewChangelog" class="btn btn-primary" title="Change Log" ng-click="getChangeLogs(user.username)">View Change Log</button>
```

Comment:
can't this just be a link to `/change-logs?username={{username()}}` instead of a button that does navigation? (`$scope.username` will need to be a function that returns the sanitized username with `encodeURIComponent` for that example to work)

Reply:
I can but what's wrong in how it is right now?

Reply:
Because click handlers should never perform navigation if possible. I can't middle-click a button to open it in a new tab, or right-click -> copy link URL, or any of the other things I expect from links. It also messes with assistive technologies; if you rely on a screen reader to find hyperlinks between pages, this can be annoying to dig through.

Reply:
Understood.